### PR TITLE
Feature 17: highlighting for types,keywords

### PR DESCRIPTION
**Pull request by BaranovDaniil**
Highlighting for types, keywords and for words after ' . ' ( it seems like highlighting for methods or atribute )

### CODE REVIEW

File: example/lib/custom_code_box.dart [hunk 1]
```diff
@@ -29,18 +29,14 @@ class _CustomCodeBoxState extends State<CustomCodeBox> {
...
-      "ruby",
+      "scala",
+      "go",
     };
     return <String?>[
       ...TOP,
-      null, // Divider
-      ...CODE_SNIPPETS.keys.where((el) => !TOP.contains(el))
+      // null, // Divider
+      // ...CODE_SNIPPETS.keys.where((el) => !TOP.contains(el))
```
Comment (GrigoriyKostryukov) [inline example/lib/custom_code_box.dart]: The same problem as in PR #3. Try not to publish the same changes to different branches

File: example/lib/custom_code_box.dart [hunk 2]
```diff
@@ -29,18 +29,14 @@ class _CustomCodeBoxState extends State<CustomCodeBox> {
   List<String?> get languageList {
     const TOP = <String>{
       "java",
-      "cpp",
       "python",
-      "javascript",
-      "cs",
-      "dart",
```
Comment (SvetlanaRyzhkova) [inline example/lib/custom_code_box.dart:36]: Add dart to this list.

File: lib/languages/all.dart [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import 'go.dart';
+import 'java.dart';
+import 'python.dart';
+import 'scala.dart';
+
+final builtinLanguages = {
+  'go': go,
+  'java': java,
+  'python': python,
```
Comment (SvetlanaRyzhkova) [inline lib/languages/all.dart:10]: Add dart

**Review by SvetlanaRyzhkova — APPROVED** [review]

File: example/lib/custom_code_box.dart (hunk 1 above)
Comment (BaranovDaniil) [reply example/lib/custom_code_box.dart]: changes are copied as the PRs depend on each other

**Review by darki-D4C — APPROVED** [review]

**Review by a-kokorev — APPROVED** [review]

**Pull request merged**